docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -206,11 +206,11 @@ Returns all information associated with the account of provided Pubkey
 fields:
   - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
   - `encoding: <string>` - encoding for Account data, either "base58" (*slow*),
-"base64", or jsonParsed". "base58" is limited to Account data of less than 128
+"base64", or "jsonParsed". "base58" is limited to Account data of less than 128
```

Comment:
Do you mind fixing this everywhere that copy-pasta has put it? (eg. `getMultipleAccounts`, `getProgramAccounts`, etc)

Reply:
Looks like there are 6 other instances of that missing quotation mark

Reply:
```
trent@farnsworth:~/code/solana/solana/docs$ git grep --count jsonParsed
src/developing/clients/jsonrpc-api.md:24
trent@farnsworth:~/code/solana/solana/docs$ git grep --count '\(^\|[^"]\)jsonParsed"' 
trent@farnsworth:~/code/solana/solana/docs$
```

Mind pointing me at one? :confused:

Reply:
https://github.com/solana-labs/solana/blame/master/docs/src/developing/clients/jsonrpc-api.md#L1560
(am I missing a good github way to link the line outside of `blame`?)

I'm not so quick on the regex, but doing a text search for ` jsonParsed"`(note space) finds them.

Reply:
I think I got all of those in https://github.com/solana-labs/solana/commit/f63231ae6c73c471fab7847e124d7c6d6c2d4ca7. Though while digging around for the missing quotes, I noticed that I'd replaced "Parsed-JSON" with "parsedJson" instead of "jsonParsed", so it's now https://github.com/solana-labs/solana/pull/13483/commits/d6c039decb1affbf914f5746143831a47473a527

> am I missing a good github way to link the line outside of blame?

You are not. However, Github _is_!

Reply:
Lol, good catch :) 
However, I'm still not seeing those missing quotation marks added. Missing a push, perhaps?